### PR TITLE
[doc]import-csv-file-cn

### PR DESCRIPTION
**Pull request by wilsonyou**

### CODE REVIEW

File: docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# CSV文件导入示例
+
+以下示例将指导您如何使用 [**Nebula Importer**](https://github.com/vesoft-inc/nebula-importer) 将CSV数据导入到 **Nebula Graph** 中。在此示例中，**Nebula Graph** 通过 `Docker` 和 `Docker Compose` 安装。我们将通过以下步骤引导您完成该示例：
```
Comment (Amber1990Zhang) [inline docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md]: space before and after CSV

File: docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+$ sudo docker pull vesoft/nebula-console:nightly
+```
+
+4. 执行以下命令以连接 **Nebula Graph** 服务器：
+
+```bash
+$ sudo docker run --rm -ti --network=host vesoft/nebula-console:nightly --addr=127.0.0.1 --port=3699
+```
+
+**注意**：您必须确保IP地址和端口号配置正确。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md]: ditto

File: docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+## 创建点和边的 Schema
+
+在输入schema之前，必须创建一个空间并使用它。在此示例中，我们创建一个 **nba** 空间并使用它。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md]: ditto

File: docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+nebula> CREATE EDGE follow (degree, int);
+```
+
+## 准备配置文件
+
+您必须配置 `.yaml` 配置文件，该文件规定了 CSV 文件中数据的格式。在本例中，我们创建一个名为 `config.yaml` 的配置文件。
+
+在此示例中，我们按以下方式配置 `config.yaml` 文件：
+
+```ngql
```
Comment (Amber1990Zhang) [inline docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md]: yaml

Comment (wilsonyou) [reply docs/manual-CN/1.overview/2.quick-start/4.import-csv-file.md]: .yaml is the file format

**Comment by dutor** [comment]
Jenkins go

**Pull request merged**